synfig-studio/src/gui/dialogs/canvasproperties.h

```diff
@@ -27,14 +27,25 @@
 
 /* === H E A D E R S ======================================================= */
 
-#include <ETL/handle>
-
+#include <gtkmm/builder.h>
+#include <gtkmm/checkbutton.h>
 #include <gtkmm/dialog.h>
-#include <gtkmm/tooltip.h>
-#include <gtkmm/table.h>
 #include <gtkmm/entry.h>
+#include <gtkmm/label.h>
+#include <gtkmm/scale.h>
+#include <gtkmm/spinbutton.h>
+#include <gtkmm/table.h>
```

Comment:
not used here with your changes AFAIK (and it's deprecated)

Reply:
Updating...

Reply:
Table is used by create_meta_data_view()

Reply:
But it shouldn't be in .cpp? It's not used in .h :)

Reply:
Got it. Moving it. :)